src/components/global/BaseTag.vue

```diff
@@ -0,0 +1,81 @@
+<script>
+import { defineComponent } from 'vue';
+
+import TagEllipse from '@/assets/icons/tag-ellipse.svg';
+
+export default defineComponent({
+    name: 'BaseTag',
+    components: {
+        TagEllipse,
+    },
+    props: {
+        labels: {
+            type: [String],
+            required: true,
+        },
+        colorScheme: {
+            validator(value) {
+                return ['red', 'gray', 'transparent'].includes(value);
+            },
+            required: true,
+        },
+    },
+});
+</script>
+
+<template>
+    <div 
+        class="base-tag"
+        :labels="labels"
+        :colorScheme="colorScheme"
+    >
+        <div class="base-tag-label-container" v-for="label in labels" :key="label">
+            <div class="base-tag-label" v-if="labels.length === 1">
+                <span class="base-tag-label-text">{{label}}</span>
+            </div>
+            <div class="base-tag-label" v-else-if="labels.length > 1">
+                <span class="base-tag-label-text">{{label}}</span>
+                <TagEllipse class="base-tag-label-ellipse"  />
+            </div>
```

Comment:
maybe it would be cleaner and shorter to add `v-if` just for `<TagEllipse .../>`?

Reply:
Totally - that's done now!

Reply:
You are right - in the API there is only one `genre` per movie, `release_date` and `length`. What would you suggest?

I don't suggest anything but there is a known principle called `YAGNI` 🙈

Reply:
Ok, I wanted to double check before moving on!

Anyway, I'm leaving the `<TagEllipse />` element just for the sake of the case where both the release year and duration are displayed in the same tag/pill.

This put aside, I'll follow Your non-suggestion and use `YAGNI` when in doubt 😅